db/R/get.trait.data.R

```diff
@@ -272,15 +272,16 @@ runModule.get.trait.data <- function(settings) {
     modeltype <- settings$model$type
     dbfiles <- settings$database$dbfiles
     database <- settings$database$bety
-    forceupdate <- !is.null(settings$meta.analysis) & settings$meta.analysis$update
+    if ((forceupdate == 'AUTO') || !as.logical(forceupdate))
+    forceupdate <- !is.null(settings$meta.analysis) && (as.logical(settings$meta.analysis$update) || (settings$meta.analysis$update == 'AUTO'))
     settings$pfts <- get.trait.data(pfts, modeltype, dbfiles, database, forceupdate)
     return(settings)
   } else if(is.Settings(settings)) {
     pfts <- settings$pfts
     modeltype <- settings$model$type
     dbfiles <- settings$database$dbfiles
     database <- settings$database$bety
-    forceupdate <- !is.null(settings$meta.analysis) &settings$meta.analysis$update
+    forceupdate <- !is.null(settings$meta.analysis) && (as.logical(settings$meta.analysis$update) || (settings$meta.analysis$update == 'AUTO'))
```

Comment:
I'm not very familiar with get.trait.data functions, but `forceupdate` is set to TRUE here for a standard pecan.xml I think, then this means this whole if chunk won't run [here](https://github.com/PecanProject/pecan/blob/db19bcadbd7fb7e67ce80a5a428acde5d7b0868e/db/R/get.trait.data.R#L86), I think we want this chunk to be run, right?